content/blog/how-to-deploy-application-across-hybrid-clouds-beyond-hpe-greenlake-for-private-cloud-enterprise.md

```diff
@@ -0,0 +1,269 @@
+---
+title: How to deploy application across hybrid clouds beyond HPE GreenLake for
+  Private Cloud Enterprise
+date: 2023-01-02T07:52:05.869Z
+author: Guoping Jia
+authorimage: /img/guoping.png
+disable: false
+---
+## Introduction
+
+[HPE GreenLake for Private Cloud Enterprise (PCE)](https://www.hpe.com/us/en/greenlake/private-cloud-enterprise.html) delivers a modern private cloud to support your app workloads with bare metal, containers, and virtual machines (VMs) running in any combination across your edges, colocations, and data centers. It combines self-service resource access for developers with consumption and performance transparency for IT. 
+
+This blog post show you how to deploy a complex application that consists of multiple microservices as a hybrid app that spans both public AWS EKS cluster and private K
+ubernetes cluster in HPE GreenLake PCE. By using a hybrid cloud solution, you can combine the compliance benefits of a private cloud in HPE GreenLake PCE environment with the scalability and connectivity of the public cloud. You can rely on the security of finely tuned, on-premises data centers while turning to the agility of cloud computing to manage the font end of an application in the public cloud. It essentially can optimize resource allocation, save costs and improve overall productivity and performance in the process. 
+
+## Prerequisites
+
+Before you start, make sure you meet the following requirements: 
+
+* A public Kubernetes cluster from one of the public cloud providers such as *AWS*, *Microsoft Azure* or *Google*. We use one EKS cluster, named *eks-cfe-public* from AWS, in this blog for application deployment. However, it works if you choose a cluster from other providers.
+* A private Kubernetes cluster, named *eks-pce-clu-1* provisioned in HPE GreenLake for Private Cloud Enterprise; 
+* The *kubectl* CLI tool, version 1.23 or later, together with the *kubeconfig* files for accessing both public and private clusters;
+* The [Skupper](https://skupper.io/start/#step-1-install-the-skupper-command-line-tool-in-your-environment) CLI tool, the latest version 1.2.0.
```

Comment:
Could you please mention here where the Skupper CLI tool needs to be installed (does it need to be installed on the reader's PC/Laptop, the public AWS EKS cluster, and/or the private K8s cluster?